<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Localizador de Texto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textoFonte">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Digite o texto onde as localizações ocorrerão.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textoLocalizar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Digite o texto a ser localizado.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_localizar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pressione este botão para realizar a pesquisa.</string>
    </property>
    <property name="text">
     <string>Localizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_limpar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Após realizar uma busca utilize este botão para poder realizar outras buscas.</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="c_caseSensitive">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fazer busca 'case sensitive'.</string>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="text">
     <string>Diferenciar letras maiúsculas e minúsculas</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_resultado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resultado: 0 ocorrência(s).</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
